integrations/hotjar/HISTORY.md

```diff
@@ -1,3 +1,8 @@
+1.2.0 / 2019-10-18
```

Comment:
@CarlosMecha Just copied this PR over from Webonise's fork. Any reason we are bumping to 1.2.0 here rather than 1.1.0?

Reply:
yeah, I think the version 1.1.0 is already out, but not documented.

Reply:
Weird, definitely not on npm: https://www.npmjs.com/package/@segment/analytics.js-integration-hotjar